### PR TITLE
User sign-up doc

### PR DESCRIPTION
**Pull request by redwarewolf**
## Summary

- Wrote some nifty documentation about signing up, it looks pretty if I may say so myself

## Known Issues

- There is a small probability that upon entering the documentation URL your hard drive gets encrypted and sold in Ebay

## Trello Card

[Documentation card](https://trello.com/c/JCWxdCiG/36-documentaci%C3%B3n-de-sign-up)

### CODE REVIEW

File: documentation/schemas/user.js [hunk 1]
```diff
@@ -3,13 +3,17 @@ module.exports = {
...
     example: 7
   },
-  username: {
+  firstName: {
     type: 'string',
-    example: 'tom99'
+    example: 'tom'
   },
-  userEmail: {
+  lastame: {
```
Comment (picollomartin) [inline documentation/schemas/user.js:10]: You have a typo here

File: documentation/paths/user.js [hunk 2]
```diff
@@ -1,33 +1,5 @@
...
-                $ref: '#/components/schemas/Users'
-              }
-            }
-          }
-        }
-      }
-    },
     post: {
       tags: ['CRUD operations'],
       description: 'Create user',
```
Comment (picollomartin) [inline documentation/paths/user.js:5]: Missing parameters definition

Comment (picollomartin) [reply documentation/paths/user.js:5]: This still applies 😅

File: documentation/paths/user.js [hunk 3]
```diff
@@ -47,16 +19,44 @@ module.exports = {
...
+              },
+              example: {
+                message: 'Password length is less than 8 characters',
+                internal_code: 'invalid_params'
+              }
+            }
+          }
+        },
+        400: {
+          description: 'Database error',
```
Comment (picollomartin) [inline documentation/paths/user.js:51]: Database error = 400 ? Are you sure ?

File: documentation/paths/user.js (hunk 2 above)
Comment (redwarewolf) [reply documentation/paths/user.js:5]: Ooops I totally missed this

**Pull request merged**